test/plugins/ExportKeras/ExportKeras.spec.js

```diff
@@ -0,0 +1,155 @@
+    const testFixture = require('../../globals'),
+        BlobClient = require('webgme-engine/src/server/middleware/blob/BlobClientWithFSBackend'),
+        {promisify} = require('util'),
+        assert = require('assert'),
+        ARCHITECTURE = testFixture.ARCHITECTURE,
+        gmeConfig = testFixture.getGmeConfig(),
+        logger = testFixture.logger.fork('ExportKeras'),
+        PluginCliManager = testFixture.WebGME.PluginCliManager,
+        manager = new PluginCliManager(null, logger, gmeConfig),
+        projectName = 'testProject',
+        pluginName = 'ExportKeras',
+        blobClient = new BlobClient(gmeConfig, logger);
+
+    let project,
+        gmeAuth,
+        storage,
+        commitHash,
+        plugin;
+
+    before(async function () {
+        gmeAuth = await testFixture.clearDBAndGetGMEAuth(gmeConfig, projectName);
+        storage = testFixture.getMemoryStorage(logger, gmeConfig, gmeAuth);
+        await storage.openDatabase();
+        const importParam = {
+            projectSeed: testFixture.testSeedPath,
+            projectName: projectName,
+            branchName: 'master',
+            logger: logger,
+            gmeConfig: gmeConfig
+        };
+        const importResult = await testFixture.importProject(storage, importParam);
+        project = importResult.project;
+        commitHash = importResult.commitHash;
+
+        await project.createBranch('test', commitHash);
+        plugin = await manager.initializePlugin(pluginName);
+        const context = {
+            project: project,
+            commitHash: commitHash,
+            branchName: 'test',
+            activeNode: ''
+        };
+        await manager.configurePlugin(plugin, {}, context);
+    });
+
+    after(async function () {
+        await storage.closeDatabase();
+        await gmeAuth.unload();
+    });
+
+    async function getGeneratedJSON(nodeId) {
+        const pluginConfig = {};
+        const context = {
+            project: project,
+            commitHash: commitHash,
+            branchName: 'test',
+            activeNode: nodeId,
+            namespace: 'keras'
+        };
+
+        let awaitableExecutePlugin = promisify(manager.executePlugin);
+
+        const result = await awaitableExecutePlugin(pluginName, pluginConfig, context);
+
+        assert.equal(result.success, true, 'Error, Plugin execution failed');
+        const jsonHash = result.artifacts[0];
+        return await blobClient.getObjectAsJSON(jsonHash);
+    }
+
+    describe('multi arch output', function () {
+        let modelJSON;
+        const layerNames = ['input_1', 'activation_1', 'dense_1', 'dropout_1'];
+        const layerClassNames = ['InputLayer', 'Activation', 'Dense', 'Dropout'];
+
+        before(async function () {
+            this.timeout(5000);
+            modelJSON = await getGeneratedJSON(ARCHITECTURE.MultiArchOutputs);
+        });
+
+        it('should match layer names and number of layers', function () {
```

Comment:
Some people/projects can be more strict and say that there should be only one `assert` per test. I don't think it has to be that rigid but a lot of asserts suggests that you may want to split it up into smaller tests. 

In this case, it seems that there are at least three things being tested:
- number of layers is correct
- layers are correct type
- layers have correct names

Reply:
Now using one assert per test.